### DISCUSSION TITLE
Has the development of the theme stopped?

### DISCUSSION THREAD

**Discussion by vkaushal288-alivecor** [mmistakes/minimal-mistakes#4265] [General]
The last release was more than 1.5 years ago. Latest commit is almost 9 months ago. I don't see much activity in the issues either. So the question obviously comes to mind - has the development of the theme stopped?

**Comment by juweins** [comment]
Unfortunately, I think so. But let's see if anyone else is responding to our discussions.

**Comment by domsalisbury** [comment]
I was wondering this too. At some point the code will likely become defunct, so I'm wondering if I should swap to a new theme or platform.
